packages/react-form/src/hooks/list/dynamiclist.ts

```diff
@@ -1,12 +1,17 @@
 import {FieldDictionary} from '../../types';
 
-import {addFieldItemAction, removeFieldItemAction} from './hooks';
+import {
+  addFieldItemAction,
+  removeFieldItemAction,
+  moveFieldItemAction,
+} from './hooks';
 import {useBaseList, FieldListConfig} from './baselist';
 
 interface DynamicList<Item extends object> {
   fields: FieldDictionary<Item>[];
   addItem(factoryArgument?: any): void;
   removeItem(index: number): void;
+  moveItem(oldIndex: number, newIndex: number): void;
```

Comment:
Just "index" or "existingIndex" could be better than "oldIndex". Have you considered calling these properties "fromIndex"+"toIndex"? (Don't know if you saw my earlier comment about that.)

Reply:
no, I didn't. I do like the `fromIndex` and `toIndex` because it's pretty informative for both args - will update